tpl/partials/integration_test.go

```diff
@@ -225,7 +225,32 @@ D1
 	b.Assert(got, hqt.IsSameString, expect)
 }
 
-//  gobench --package ./tpl/partials
+func TestP(t *testing.T) {
+	t.Parallel()
+
+	// TODO(bep) also add some error cases (missing partial, etc.)
+	files := `
+-- config.toml --
+baseURL = 'http://example.com/'
+-- layouts/index.html --
+{{ "abcd" | p.funcs.ToUpper }}
```

Comment:
@regisphilibert this looks good to me.

Reply:
Great... I added some last reflexions on #10544 which could be addressed later I suppose.